src/public/navigation.ts

```diff
@@ -13,7 +13,7 @@ import { sendMessageToParent } from '../internal/communication';
  * @param navigateForward Determines the direction to focus in teams app.
  */
 export function returnFocus(navigateForward?: boolean): void {
-  ensureInitialized(FrameContexts.content);
+  ensureInitialized();
```

Comment:
Are you trying to add meeting stage context only? Or do you want this API to work in all contexts?

Reply:
Doing it for all now as per comments from Yuri